PixelHunter1995/Screen.cs

```diff
@@ -1,27 +1,33 @@
 ﻿using Microsoft.Xna.Framework;
 using Microsoft.Xna.Framework.Graphics;
-using Microsoft.Xna.Framework.Input;
 using PixelHunter1995.Utilities;
+using PixelHunter1995.Inputs;
 
 namespace PixelHunter1995
 {
     /** This class is responsible for switching between windowed and fullscreen mode.
      */
     internal class Screen
     {
+
+        public Rectangle renderTargetRect;
+        public int Width { get; private set; }
+        public int Height { get; private set; }
+
         internal int fullScreenWidth;
         internal int fullScreenHeight;
-        public Rectangle renderTargetRect;
+
         private readonly GraphicsDeviceManager graphics;
         private readonly GameWindow window;
-        private bool justToggledFullscreen = false;
 
         public Screen(GraphicsDeviceManager graphics, GameWindow window)
         {
             this.graphics = graphics;
             this.window = window;
-            graphics.PreferredBackBufferWidth = GlobalSettings.WINDOW_WIDTH;
-            graphics.PreferredBackBufferHeight = GlobalSettings.WINDOW_HEIGHT;
+            this.Width = GlobalSettings.WINDOW_WIDTH;
```

Comment:
why save? you can just read GLobalSettings again

Reply:
Because FullscreenFix needs access to the current setting in `graphics.PreferredBackBuffer.....?` but has no reference to graphics. Figured it was best to expose it through Screen as two properties Width and Height.